templates/cmake.mk

```diff
@@ -26,12 +28,11 @@ COMPONENT_ARCHIVE_URL=
 COMPONENT_ARCHIVE_HASH=
 COMPONENT_LICENSE=
 
-include $(WS_MAKE_RULES)/prep.mk
-include $(WS_MAKE_RULES)/cmake.mk
-include $(WS_MAKE_RULES)/ips.mk
+TEST_TARGET=$(NO_TESTS) # if no testsuite enabled
```

Comment:
Does this work as expected? Shouldn't TEST_TARGET only be set if not yet set?

Reply:
This has the same effect as `test: $(NO_TESTS)` which was previously in the file. Do you intend to change the default?

Reply:
No, I misunderstood how it works